.github/workflows/release.yaml

```diff
@@ -16,16 +16,12 @@ jobs:
     defaults:
```

Comment:
The release workflow now only publishes packages if the event name is 'release'.  Previously, it would publish regardless of the triggering event. This change prevents accidental publishing of packages during other GitHub events (e.g., pull requests). However, it also means that non-release events will no longer trigger publishing, which might be a desired behavior depending on the project's CI/CD strategy.  If publishing on other events is needed, the `if` condition should be removed or modified.